pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -387,6 +387,9 @@ class Build {
                         }
 
                         def additionalTestLabel = buildConfig.ADDITIONAL_TEST_LABEL
+                        def relatedNodeLabel = buildConfig.RELATED_NODES
+                        def jenkinsKey = buildConfig.JENKINS_KEY
```

Comment:
`jenkinsKey` should not be defined in script. It should be set in Test_Job_Auto_Gen.

Reply:
Test_Job_Auto_Gen job config has been updated with default value for JENKINS_KEY.